src/indicators/ipc_food_insecurity/update_food_insecurity.R

```diff
@@ -7,6 +9,17 @@ box::use(./utils/map_food_insecurity)
 box::use(./utils/info_food_insecurity)
 
 box::use(../../alerts/generate_signals[generate_signals])
+box::use(../../utils/logger)
+
+logger$configure_logger()
+
+test <- as.logical(Sys.getenv("TEST", unset = FALSE))
+test_filter <- if (test) c("AFG", "SSD") else NULL
+
+log_info("Checking food insecurity indicator...")
+log_debug(paste0("GMAS_TEST_RUN = ", Sys.getenv("GMAS_TEST_RUN")))
```

Comment:
Don't think we'd want to use `get_env` here since `GMAS_TEST_RUN` doesn't necessarily need to be set. 

And yeah, I like that to simplify things further!